qutebrowser/browser/downloads.py

```diff
@@ -364,6 +370,16 @@ def cancel(self, remove_data=True):
         self.finished.emit()
         self.data_changed.emit()
 
+    def path_suggestion(self):
+        """Get the suggestion of file path"""
+        suggestion = config.get('completion', 'download-path-suggestion')
+        if suggestion == 'path':
+            return self._download_dir()
+        elif suggestion == 'filename':
+            return self.basename
+        else:
```

Comment:
Manual changes actually won't be a problem, the config will be validated according to  `configtypes.DownloadPath`'s `valid_values`.

It's more if there was some bug like someone adding a new value to that but forgetting to add it here - unlikely to happen here, but if something like that happens, failing loudly is better than silently assuming a value ;)